backend/local_graphql_server.py

```diff
@@ -8,9 +8,9 @@
 
 from dataall.base.api import get_executable_schema
 from dataall.core.tasks.service_handlers import Worker
-from dataall.core.permissions import permissions
-from dataall.core.permissions.db import save_permissions_with_tenant
-from dataall.core.permissions.db.tenant_policy_repositories import TenantPolicy
+from dataall.core.permissions.constants import permissions
```

Comment:
I still think that we should define the permissions in the service layer not in constants to keep it consistent to how we do it across the repo. Plus they are not constants, permissions are a central piece of the business logic

Reply:
In addition, i would import TENANT_ALL permissions only